### ISSUE TITLE
Can't create an admin user in MongoDB

### ISSUE THREAD

**Issue by virshu** [angular-app/angular-app#261]
I am following the readme instructions, and everything goes fine until I reach "Run our initialization script to initialize the database with a first admin user" step in configure server section. I am getting the following error when I run server/initDB.js:
...
Created new admin user...
{ [Error: getaddrinfo ENOTFOUND] code: 'ENOTFOUND', errno: 'ENOTFOUND', syscall: 'getaddrinfo' }
undefined

This is my first experience with MongoDB, so I am probably missing something obvious; however...

When I create a user from Mongo console, it asks for id, password and whether it is a read-only account. Very different from email / admin / etc. that is passed in initDB script.

I created an account manually; however, logging in through SCRUM app gives me "wrong name/password" error.

**Comment by holladayinnn** [comment]
Thank you.  adding the '/' at the end of 'https://api.mongolab.com/api/1' worked for me too.

**Comment by anujmca** [comment]
I am constantly getting this error "Error: Please provide a valid API key." 

I have tried all the solutions mentioned here but nothing seems to be working, may I please request some help here.

**Comment by hosszubalazs** [comment]
Hey @anujmca !

Login to Mongolabs, and go to the following page: 
https://mlab.com/user?username=[YOUREUSERNAME].

This is the page where you can check your API key and generate a new one. At the bottom of the page, click "Enable Data API access". I had a similar problem like you and this solved it for me. I also generated a new API key after enabling, this is probably not necessary.

Hope this helps!

**Comment by virshu** [comment]
@anujmca - I never got the error you got, and the solution that worked for me was to add trailing slash. Look - I don't mind you hijacking this thread two years later with completely irrelevant question, but if you were to create a separate thread for your question, you may have got more responses.
[http://www.urbandictionary.com/define.php?term=Thread+Hijacking](url)

**Comment by anujmca** [comment]
Solution by "hubaloo" worked, it was "Enable Data API access" which did the trick. Thank you.

**Comment by imustak07** [comment]
Yes, "Enable Data API access" and it will work. Otherwise it throws "Error: Please provide a valid API key."

**Comment by godzillalad** [comment]
I had this problem connecting on an Android device and found that it was the encoding of the parameters

handy function `URI.parse` took care of that

`
String uri = Uri.parse("https://api.mlab.com/api/1/databases/" + MLAB_DATABASE + "/collections/" + MLAB_COLLECTION)
                        .buildUpon()
                        .appendQueryParameter("q", "{\"Id\":" + 123+ "}")
                        .appendQueryParameter("apiKey", MLAB_API_KEY)
                        .build().toString();

              
URL urlToRequest = new URL(uri);
urlConnection = (HttpURLConnection) urlToRequest.openConnection();`
